musi/pk.srt

```diff
@@ -2595,207 +2595,271 @@ please.
 
 593
 01:11:39,429 --> 01:11:41,208
-hey why you locking?
+sina pini e lupa tan seme?
+==hey why you locking?
 
 594
 01:11:41,254 --> 01:11:42,445
-i won't let you go today.
+tenpo suno ni la sina tawa ala.
+==i won't let you go today.
 
 595
 01:11:42,842 --> 01:11:45,104
-open the door, i have an entry.
+o open e lupa. mi wile tawa.
+==open the door, i have an entry.
 
 596
 01:11:45,540 --> 01:11:49,224
+sewi Siwa. ona li pakala e ike
+li monsuta e ala.

```

Comment:
For me "pilin akesi" fits. "pilin ..." in general is a very strange construction used in Toki Pona. (I will have to think about it one more time.) But yes, "pilin ike" is fine.